debezium-core/src/main/java/io/debezium/relational/history/KafkaDatabaseHistory.java

```diff
@@ -149,6 +160,39 @@ public void configure(Configuration config, HistoryRecordComparator comparator)
     @Override
     public synchronized void start() {
         super.start();
+        createDatabaseHistoryTopic();
+    }
+
+    private void createDatabaseHistoryTopic() {
+        final AdminClient admin = AdminClient.create(this.producerConfig.asProperties());
+        try {
+            short replicationFactor;
+
+            Optional<String> findTopic = admin.listTopics().names().get(KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS).stream().filter(x -> topicName.equals(x)).findFirst();
+
+            if (!findTopic.isPresent()) {
+                // Find default replication factor
+                final Collection<Node> nodes = admin.describeCluster().nodes().get(KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
+                if (nodes.isEmpty()) {
+                    throw new ConnectException("No brokers available to obtain default settings");
+                }
+                final Map<ConfigResource, Config> configs = admin.describeConfigs(Collections.singleton(new ConfigResource(ConfigResource.Type.BROKER, nodes.iterator().next().idString()))).all().get(KAFKA_QUERY_TIMEOUT.toMillis(), TimeUnit.MILLISECONDS);
+                if (configs.isEmpty()) {
+                    throw new ConnectException("No configs has arrived");
+                }
+                final Config config = configs.values().iterator().next();
+                replicationFactor = Short.parseShort(config.get("default.replication.factor").value());
+
+                // Create topic
+                final NewTopic topic = new NewTopic(topicName, (short)1, replicationFactor);
+                topic.configs(Collect.hashMapOf("cleanup.policy", "delete", "retention.ms", Long.toString(Duration.ofDays(365 * 10).toMillis())));
+                admin.createTopics(Collections.singleton(topic));
+            } else {
+                logger.info("Database history topic exists");
+            }
+        } catch (Exception e) {
+            logger.warn("Could not manually create databse history topic, delegating to admin or broker", e);
```

Comment:
databse=database

Also "manually" is confusing, isn't it the opposite? i'd call it "manually" if the user had to create the topic themselves".